buildspec.yml

```diff
@@ -0,0 +1,42 @@
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      nodejs: 14
```

Comment:
Should we upgrade nodejs to the latest avaible version of the build environment?